entity-service/src/main/kotlin/com/egm/stellio/entity/model/Property.kt

```diff
@@ -69,15 +69,20 @@ class Property(
         return propsMap
     }
 
-    fun updateValues(unitCode: String?, value: Any?, observedAt: ZonedDateTime?) {
+    fun updateValues(unitCode: String?, value: Any?, observedAt: ZonedDateTime?): Int {
+        var updateResult = 0
```

Comment:
This `updateResult` variable does not make sense (the update does not occur here)

Reply:
The reason behind declaring this variable is to return the number of core properties updated.